src/build/__main__.py

```diff
@@ -160,7 +160,7 @@ def main_parser():  # type: () -> argparse.ArgumentParser
         '-C',
         action='append',
         help='pass options to the backend.  options which begin with a hyphen must be in the form of '
-        '`--config-setting=--opt(=value)` or `-C--opt(=value)`',
+        '``--config-setting=--opt(=value)`` or ``-C--opt(=value)``',
```

Comment:
@FFY00 this is an alternative solution:
```suggestion
        '"--config-setting=--opt(=value)" or "-C--opt(=value)"',
```
(you can accept the change and merge it if you prefer it or just leave it as is, that's up to you)